### PR TITLE
feat: add single project example

### PR DESCRIPTION
**Pull request by amandakarina**
Add Single Project example, integration tests and E2E tests.

### CODE REVIEW

File: 2-multitenant/modules/env_baseline/main.tf
Comment (apeabody) [inline 2-multitenant/modules/env_baseline/main.tf:220]: nit
```suggestion
    module.eab_cluster_project, 
    google_project_iam_member.gke_service_agent, 
    google_project_iam_member.servicemesh_service_agent, 
    google_project_iam_member.multiclusterdiscovery_service_agent
```

File: 2-multitenant/modules/env_baseline/main.tf
Comment (apeabody) [inline 2-multitenant/modules/env_baseline/main.tf:265]: nit

```suggestion
    module.eab_cluster_project, 
    google_project_iam_member.gke_service_agent, 
    google_project_iam_member.servicemesh_service_agent, 
    google_project_iam_member.multiclusterdiscovery_service_agent
```

File: 3-fleetscope/modules/env_baseline/mcg.tf
Comment (apeabody) [inline 3-fleetscope/modules/env_baseline/mcg.tf:32]: likely need to remove the comma

```suggestion
    google_gke_hub_feature.mcs
```

File: 3-fleetscope/modules/env_baseline/mcg.tf
Comment (apeabody) [inline 3-fleetscope/modules/env_baseline/mcg.tf:49]: same

```suggestion
    google_gke_hub_feature.mci
```

File: 2-multitenant/modules/env_baseline/main.tf
Comment (apeabody) [inline 2-multitenant/modules/env_baseline/main.tf:113]: optional: could use a for_each to combine

File: test/integration/standalone_single_project/standalone_single_project_test.go
Comment (apeabody) [inline test/integration/standalone_single_project/standalone_single_project_test.go:42]: Are we seeing drift?

File: test/integration/standalone_single_project/standalone_single_project_test.go
Comment (apeabody) [inline test/integration/standalone_single_project/standalone_single_project_test.go:47]: not sure this comment is for here?

**Comment by apeabody** [comment]
Hey @amandakarina - Not sure why the lint check didn't trigger for this PR, might want to check locally.

File: 5-appinfra/modules/cicd-pipeline/project-iam-bindings.tf
Comment (apeabody) [inline 5-appinfra/modules/cicd-pipeline/project-iam-bindings.tf:30]: Just curious if `"additive"` would have worked.

File: 5-appinfra/modules/cicd-pipeline/variables.tf
Comment (apeabody) [inline 5-appinfra/modules/cicd-pipeline/variables.tf:26]:
```suggestion
  description = "Cluster services accounts to be granted the Artifact Register reader role."
```

File: examples/standalone_single_project/outputs.tf
Comment (apeabody) [inline examples/standalone_single_project/outputs.tf:17]: duplicate header

File: examples/standalone_single_project/outputs.tf
Comment (apeabody) [inline examples/standalone_single_project/outputs.tf:96]:
```suggestion
}

```

File: test/integration/standalone_single_project/standalone_single_project_test.go
Comment (amandakarina) [reply test/integration/standalone_single_project/standalone_single_project_test.go:42]: Got a cycle error:

> Error: Invalid for_each argument
        	            	
        	            	  on .terraform/modules/cicd.project-iam-bindings/modules/projects_iam/main.tf line 52, in resource "google_project_iam_member" "project_iam_additive":
        	            	  52:   for_each = module.helper.set_additive
        	            	    ├────────────────
        	            	    │ module.helper.set_additive is a set of dynamic, known only after apply
        	            	
        	            	The "for_each" set includes values derived from resource attributes that
        	            	cannot be determined until apply, and so Terraform cannot determine the full
        	            	set of keys that will identify the instances of this resource.
        	            	
        	            	When working with unknown values in for_each, it's better to use a map value
        	            	where the keys are defined statically in your configuration and where only
        	            	the values contain apply-time results.
        	            	
        	            	Alternatively, you could use the -target planning option to first apply only
        	            	the resources that the for_each value depends on, and then apply a second
        	            	time to fully converge.

File: 5-appinfra/modules/cicd-pipeline/project-iam-bindings.tf
Comment (amandakarina) [reply 5-appinfra/modules/cicd-pipeline/project-iam-bindings.tf:30]: Got a cycle error:

> Error: Invalid for_each argument
        	            	
        	            	  on .terraform/modules/cicd.project-iam-bindings/modules/projects_iam/main.tf line 52, in resource "google_project_iam_member" "project_iam_additive":
        	            	  52:   for_each = module.helper.set_additive
        	            	    ├────────────────
        	            	    │ module.helper.set_additive is a set of dynamic, known only after apply
        	            	
        	            	The "for_each" set includes values derived from resource attributes that
        	            	cannot be determined until apply, and so Terraform cannot determine the full
        	            	set of keys that will identify the instances of this resource.
        	            	
        	            	When working with unknown values in for_each, it's better to use a map value
        	            	where the keys are defined statically in your configuration and where only
        	            	the values contain apply-time results.
        	            	
        	            	Alternatively, you could use the -target planning option to first apply only
        	            	the resources that the for_each value depends on, and then apply a second
        	            	time to fully converge.

Comment (apeabody) [reply 5-appinfra/modules/cicd-pipeline/project-iam-bindings.tf:30]: Interesting

Comment (amandakarina) [reply 5-appinfra/modules/cicd-pipeline/project-iam-bindings.tf:30]: https://console.cloud.google.com/cloud-build/builds/b4163550-4e82-43dd-b448-2ada7a0a6f15;step=24?project=cloud-foundation-cicd

**Review by apeabody — APPROVED** [review]
Thanks @amandakarina! LGTM

@gtsorbo?

File: 5-appinfra/modules/cicd-pipeline/variables.tf
Comment (caetano-colin) [inline 5-appinfra/modules/cicd-pipeline/variables.tf:25]: tought: this can be named a more generic variable, like additional_service_accounts or addicionail_artifact_registry_members

**Comment by apeabody** [comment]
@gtsorbo - CI is green, just waiting on your final review!

**Review by gtsorbo — APPROVED** [review]
Thank you!

**Pull request merged**
